storage/README.md

```diff
@@ -0,0 +1,4 @@
+collection of different storage interfaces that are shared by two or more components.
```

Comment:
proper english is good english, make sure you update the comment in storage/doc.go as well.